### ISSUE TITLE
Any plans for a new release

### ISSUE THREAD

**Issue by koutheir** [vagnum08/cpupower-gui#112]
This project is great and very useful, but the last release is 3 years old. Are there any plans to make a new release?

**Comment by pramodhrachuri** [comment]
Considering that the author, vagnum08, doesn't have any activity on their GitHub (including other repos), we can consider this project as abandoned. 

Would you happen to know of any alternatives? This project works very well even with amd_pstate enabled.

**Comment by koutheir** [comment]
> Would you happen to know of any alternatives?

Not really. I'll continue using this application, until it breaks, eventually.

**Comment by MasterCATZ** [comment]
shame it really needs to list the coreid's to make it easy to switch between allowing threading and just using the cores 
and no boost switch 
might be time to make a fork , 
I was using CoreCtl  and konkor/cpufreq

my new pc will be an epyc but its boosting is power limited so wanting to disable threading to allow higher core speeds 
at least in all my testing when ever I have threading enabled cpu draws more power / higher temps  with negligible performance increase , 128 cores
